developer-docs-site/docs/guides/get-test-funds.md

```diff
@@ -0,0 +1,67 @@
+## Create and Fund Aptos Account
+
+For testing purposes, you will want to create an Aptos account and fund it with testnet tokens. This document accompanies the command line instructions for the Aptos CLI on [creating](../cli-tools/aptos-cli-tool/use-aptos-cli.md#initialize-local-configuration-and-create-an-account) and [funding]()(../cli-tools/aptos-cli-tool/use-aptos-cli.md#fund-an-account-with-the-faucet) accounts, focusing upon the testnet network and leveraging a graphical wallet rather than the CLI.
+
+## Prerequisites
+
+You will need these installed to proceed:
+
+* The wallet of your choice; we use the [Petra Wallet](../../guides/install-petra-wallet.md) Chrome extension.
```

Comment:
We should probably also link to https://github.com/aptos-foundation/ecosystem-projects#wallets.  Sometimes it feels like it would be nice to have a copyable blurb for things like this